programs/server/config.d/create_logs_at_startup.yaml

```diff
@@ -0,0 +1 @@
+../../../tests/config/config.d/create_logs_at_startup.yaml
```

Comment:
Maybe it wasn't supposed to be committed?

Reply:
No, it is supposed to be committed, 
similarly to other symlinks, it is needed to make local development easier.